--- qtbase/src/3rdparty/zlib/src/zutil.h.orig	2024-11-23 21:22:22
+++ qtbase/src/3rdparty/zlib/src/zutil.h	2025-04-08 19:19:07
@@ -145,22 +145,8 @@
 #  endif
 #endif
 
-#if defined(MACOS) || defined(TARGET_OS_MAC)
+#if defined(MACOS)
 #  define OS_CODE  7
-#  ifndef Z_SOLO
-#    if defined(__MWERKS__) && __dest_os != __be_os && __dest_os != __win32_os
-#      include <unix.h> /* for fdopen */
-#    else
-//     We need to include stdio.h here because zlib.h will include TargetConditionals.h
-//     This will define TARGET_OS_MAC that leads to this check.
-//     Since zutil.h will include gzguts.h and gzguts.h includes stdio.h
-//     AFTER check for fdopen we need to include stdio.h directly
-#      include <stdio.h>
-#      ifndef fdopen
-#        define fdopen(fd,mode) NULL /* No fdopen() */
-#      endif
-#    endif
-#  endif
 #endif
 
 #ifdef __acorn
